tests/ha/ha_cluster_crash_test.pm

```diff
@@ -68,12 +71,28 @@ sub run {
             }
             sleep 4;
         }
+
+        if ($check eq "kill-pacemakerd") {
+            if ($node_was_fenced == 1) {
+                record_info('ERROR', "Failure while executing '$cmd'. Killing pacemaker should not trigger fencing", result => 'fail');
+                # Wait for fencing delay and resources to start
+                sleep $fencing_start_delay;
+                wait_until_resources_started();
```

Comment:
```suggestion
                wait_until_resources_started(timeout => $fencing_start_delay);
```

Reply:
Perhaps we could try `wait_until_resources_started()` didn't work for me with the `timeout` parameter.

Reply:
the 'sleep $fencing_start_delay; ' is a delay which happens after node is fenced and boots up. The resources are not started immediately but after this time in case the node boots quicker that cluster finishes fencing procedure. It's a dead time and 'wait_after_resources_started' should begin afterwards.

I see that 'wait_after_resources_started' has default timeout of 120s but I can do a test run for that as well. 
I realized I can move this section to the grub check loop so the code is not duplicated. 

I will do another VR and ping you later.